src/index.html

```diff
@@ -6,13 +7,74 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div class="container">
+      <section class="section">
+        <article
+          data-qa="card"
+          class="product-card"
+        >
+          <div class="product-card__img">
+            <img
+              class="product-card__img_flexed"
+              src="./images/imac.jpeg"
+              alt="Apple-Imac"
+            />
+          </div>
+
+          <div class="product-card__description">
+            <h2 class="product-card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+
+            <p class="product-card__code">Product code: 195434</p>
+
+            <div class="product-card__rate">
+              <div class="product-card__stars">
+                <ul class="product-card__stars_flex">
+                  <li class="product-card__star"></li>
+                  <li class="product-card__star"></li>
+                  <li class="product-card__star"></li>
+                  <li class="product-card__star"></li>
+                  <li class="product-card__star"></li>
+                </ul>
+                <div class="product-card__reviews">Reviews: 5</div>
+              </div>
```

Comment:
You have not completed @tiserett comment - you need not only to copy check-list but to actually complete all items

For instance
> All stars--N modifiers work as expected (Highlight first N stars)

And per requirements
> Rewrite stars block from the [Stars task](https://github.com/mate-academy/layout_stars) with SCSS and use it